### PR TITLE
add note about default protocol of spring boot starter

### PR DESCRIPTION
**Pull request by zeitlinger**
Fixes https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/10943

---

**Preview**: https://deploy-preview-4258--opentelemetry.netlify.app/docs/languages/java/automatic/spring-boot/#general-configuration

### CODE REVIEW

**Review by trask — COMMENTED** [review]
Can we update the source where the defaults are listed instead, to mention this for both the Java agent and Spring boot starter?

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 1]
```diff
@@ -165,7 +165,16 @@ IDE.
 
 The OpenTelemetry Starter supports all the
 [SDK Autoconfiguration](/docs/languages/java/automatic/configuration/#sdk-autoconfiguration)
-(since 2.2.0). You can set properties in the `application.properties` or the
+(since 2.2.0). 
+
+{{% alert title="Attention" color="warning" %}}
```
Comment (chalin) [inline content/en/docs/languages/java/automatic/spring-boot.md:170]: While "Attention" would make sense in some languages other than English, I'd suggest using either "Caution", "Important", "Note", or simply "Warning"; e.g.:

```suggestion
{{% alert title="Important" color="warning" %}}
```

**Review by theletterf — APPROVED** [review]

**Comment by svrnm** [comment]
@open-telemetry/java-approvers PTAL

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 2]
```diff
@@ -165,8 +165,17 @@ IDE.
...
 The OpenTelemetry Starter supports all the
 [SDK Autoconfiguration](/docs/languages/java/automatic/configuration/#sdk-autoconfiguration)
-(since 2.2.0). You can set properties in the `application.properties` or the
-`application.yaml` file, or use environment variables.
+(since 2.2.0).
+
+{{% alert title="Attention" color="warning" %}}
+
+Unlike the SDK autoconfiguration, the OpenTelemetry Spring Boot starter uses
+`http/protobuf` as the default protocol, not `grpc`.
```
Comment (trask) [inline content/en/docs/languages/java/automatic/spring-boot.md:173]: Can we update the source where the defaults are listed instead? and to mention there the defaults for both the Java agent and Spring boot starter?

Comment (zeitlinger) [reply content/en/docs/languages/java/automatic/spring-boot.md]: done

File: content/en/docs/languages/java/automatic/configuration.md [hunk 3]
```diff
@@ -30,6 +30,13 @@ specific portions of the SDK & agent:
 - [Span limits](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#span-limits)
 - [Using SPI to further configure the SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#customizing-the-opentelemetry-sdk)
 
+{{% alert title="Important" color="warning" %}}
+
+Unlike the SDK autoconfiguration, the Java agent and OpenTelemetry Spring Boot
```
Comment (trask) [inline content/en/docs/languages/java/automatic/configuration.md:35]:
```suggestion
Unlike the SDK autoconfiguration, versions 2.0+ of the Java agent and OpenTelemetry Spring Boot
```

**Review by trask — APPROVED** [review]

**Pull request merged**